build/dist

```diff
@@ -247,14 +247,15 @@ echo -e "\$ ${BUILD_COMMAND[@]}\n"
 rm -rf "$DISTDIR"
 mkdir -p "$DISTDIR/pid"
 mkdir -p "$DISTDIR/logs"
-mkdir -p "$DISTDIR/jars"
 mkdir -p "$DISTDIR/work"
+mkdir -p "$DISTDIR/jars"
+mkdir -p "$DISTDIR/beeline-jars"
+mkdir -p "$DISTDIR/web-ui"
```

Comment:
always create this dir to avoid docker build error

```
=> ERROR [stage-2 9/9] COPY web-ui /opt/kyuubi/web-ui                                                                                     0.0s
------
 > [stage-2 9/9] COPY web-ui /opt/kyuubi/web-ui:
------
failed to compute cache key: failed to calculate checksum of ref b10njb8sgo93k7ydi5t4l84n9::9rg2u6l3o0yrkrq0ciskmfbbz: "/web-ui": not found
Failed to build Kyuubi JVM Docker image, please refer to Docker build output for details.
```